docs/rpkg/gui/opening and finding files.md

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 2
+---
+
+# Opening RPKG's
+Once setup you can load in the chunk files into the tool by using `Import` > `Import RPKGs folder` and pointing it to the `HITMAN 3\Runtime` folder. This will load all the chunk files into the tool which is recommended when you are unsure where the file you are looking for is located. You can also load in individual files using the `Import` > `Import RPKG file`
+
+# Searching RPKG's
+You can search for files using the Search tab in the left panel. Search can be done based on hash in the `Seach Hash List` tab or on filename in the `Search RPKGs` tab. The search will only search in the currently loaded RPKGs. Wildcards are not permitted.  
+
+
+
+The wiki is currently a work in progress at the moment, please feel free to contribute @ https://github.com/glacier-modding/wiki.notex.app
```

Comment:
Presumably, this comment is made obsolete by this PR.

Reply:
I'd say leave it for now in case anyone has something to add we might have overlooked.